### PR TITLE
Live streaming 404test

### PR DESCRIPTION
**Pull request by Calanthe**
Another 404 AB test variant - this time with live streaming option (including old 404 test cleanup)

Desktop:
screen shot 2015-10-21 at 16 30 11

screen shot 2015-10-21 at 16 36 44

Mobile:
screen shot 2015-10-21 at 16 30 33

screenshot

In order to pop into the test locally:
1. turn on the ab-live-events-survey
2. add `#ab-LiveEventsSurvey=variant` to the url

### CODE REVIEW

**Comment by jbreckmckye** [comment]
Maybe some of the images could be compressed further? e.g. live-streaming-survey.png is 57kb.

**Comment by Jholder112233** [comment]
That Peep Show event is on November 5th not October 5th :innocent:

File: commercial/app/views/static/adFreeSurveyPage.scala.html [hunk 1]
```diff
@@ -44,15 +44,15 @@
 
         <div class="l-side-margins">
 
-            <div class="adfreesurvey-wrapper" data-link-name="adfree survey page">
+            <div class="adfreesurvey-wrapper" data-link-name="live events survey page">
```
Comment (jbreckmckye) [inline commercial/app/views/static/adFreeSurveyPage.scala.html]: As this is no longer (as I understand) anything to do with adfree anymore, would we not want to get rid of all these references to adfree in this code? e.g. the 'adfreesurvey-wrapper' and the file name 'adFreeSurveyPage'?

**Comment by jbreckmckye** [comment]
Other than the comments, this looks fine

**Comment by paperboyo** [comment]
Great call @jbreckmckye! Here are the images... 85KB shaved. [INTERNAL RESOURCE] https://drive.google.com/open?id=0BxB_YnI6SM7jTGRpdmtMQnpOR1U

**Comment by Calanthe** [comment]
Thank you @paperboyo !

**Pull request merged**